### ISSUE TITLE
A problem of training data

### ISSUE THREAD

**Issue by XinhaoMei** [WangHelin1997/DCASE2021_Task6_PKU#1]
Hi, to achieve the performance reported in your paper, have you used the validation set of Clotho to train the model or just the training set?
Thank you.
